business/workloads.go

```diff
@@ -1371,7 +1371,11 @@ func fetchWorkload(ctx context.Context, layer *Layer, criteria WorkloadCriteria)
 		}
 		var err error
 		if isWorkloadIncluded(kubernetes.DaemonSetType) {
-			ds, err = layer.k8s.GetDaemonSet(criteria.Namespace, criteria.WorkloadName)
+			if IsNamespaceCached(criteria.Namespace) {
```

Comment:
This missing cached was being hit several times in cascade.
I had also in my config the other workloads that don't use informers and I accumulated this long delay.

The delay was always there, but with the refactoring, it was more evident.